lavamoat/browserify/experimental/policy.json

```diff
@@ -1097,6 +1097,20 @@
         "uuid": true
       }
     },
+    "@metamask/seedless-onboarding-controller>@metamask/auth-network-utils": {
```

Comment:
Hey @MetaMask/policy-reviewers, I need clarification on these `lavamoat/browserify/experimental/policy.json` updates as they appear **completely unrelated** to adding `cssnano` as a dependency for CSS minification changes.

New Runtime Packages with Powerful APIs:
- `@metamask/seedless-onboarding-controller` - Seedless wallet onboarding
- `@metamask/toprf-secure-backup` - Threshold OPRF secure backup  
- `jsonwebtoken` - JWT token handling
- Various `@toruslabs/*` packages - Web3 infrastructure

Concerning Permissions Granted:
- **Network Access**: `fetch`, `URL` construction
- **Cryptographic**: `crypto`, `msCrypto` 
- **Timing**: `setTimeout`, `clearTimeout`

Could someone help clarify if this is a merge conflict or if these changes should be in a separate PR?

Reply:
No longer updating dependencies please disregard